### PR TITLE
dynamically load leaflet only on the meetups page

### PR DESCRIPTION
**Pull request by MalayaliRobz**
fixes https://github.com/ember-learn/ember-website/issues/396
 
Results from ember build --prod before and after:
When not dynamically loading leaflet: vendor.js size: 2.61 MB (786.46 KB gzipped)
When dynamically loading leaflet: vendor.js size: 2.47 MB (747.67 KB gzipped)

### CODE REVIEW

**Comment by mansona** [comment]
Sorry for adding a commit to your PR, I was just trying to kick off the github action (apparently there are issues with it not working from a fork 😖 ) 

Here is the output anyway: 

Files that got Bigger 🚨:

File | raw | gzip
--- | --- | ---
vendor.js|+141 kB|+39.5 kB

Files that got Smaller 🎉:

File | raw | gzip
--- | --- | ---
auto-import-fastboot.js|-141 kB|-40 kB
ember-website.js|-409 B|-137 B


Files that stayed the same size 🤷‍:

File | raw | gzip
--- | --- | ---
ember-website-fastboot.js| 0 B| 0 B
ember-website.css| 0 B| 0 B
vendor.css| 0 B| 0 B

**Comment by MalayaliRobz** [comment]
> Sorry for adding a commit to your PR, I was just trying to kick off the github action (apparently there are issues with it not working from a fork 😖 )
> 
> Here is the output anyway:
> 
> Edit: it seems like the values are inverted 🙈 I will have to double and triple check this build to make sure that the size difference is correct. Sorry about the delay 😫
> 
> Files that got Bigger 🚨:
> 
> File	raw	gzip
> vendor.js	+141 kB	+39.5 kB
> Files that got Smaller 🎉:
> 
> File	raw	gzip
> auto-import-fastboot.js	-141 kB	-40 kB
> ember-website.js	-409 B	-137 B
> Files that stayed the same size 🤷‍:
> 
> File	raw	gzip
> ember-website-fastboot.js	0 B	0 B
> ember-website.css	0 B	0 B
> vendor.css	0 B	0 B

@mansona  I think i fugured out why the inverted results are coming. Looks like the diff comparison in ember-asset-size-action is inverted. Have a fix for that here: https://github.com/simplabs/ember-asset-size-action/pull/1

File: package.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
     "ember-fetch": "^6.5.1",
     "ember-group-by": "^0.0.5",
     "ember-href-to": "^2.0.1",
-    "ember-leaflet": "^3.0.17",
+    "ember-leaflet": "^4.0.2",
```
Comment (MalayaliRobz) [inline package.json]: updating to fix issue https://github.com/miguelcobain/ember-leaflet/issues/366

**Comment by MelSumner** [comment]
Ok updating with what the holdup is, now that I am aware: 

- revert #421 
- re-implement what 421 was trying to do 
- and then merge 430

the issue is that 430 depends on `ember-auto-import` which seems to have accidentally imported too much stuff

**Comment by mansona** [comment]
I cherry-picked your work in #528 targeting the redesign branch 👍

**Pull request closed without merging**